settings.gradle.kts

```diff
@@ -44,6 +43,23 @@ include(":samples:slack")
 include(":samples:static-server")
 include(":samples:tlssurvey")
 include(":samples:unixdomainsockets")
-include(":android-test")
+
+if (isIdea20232OrHigher()) {
+  include(":okhttp-android")
+  include(":android-test")
+}
 
 enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
+
+fun isIdea20232OrHigher(): Boolean {
+  // No problem outside Idea
+  val ideaVersionString = System.getProperty("idea.version") ?: return true
```

Comment:
This function should probably be inverted. It’s surprising that Gradle builds run outside IntelliJ return true, given this function’s name.

Maybe it could be isKnownBrokenIntellij()